deployment/desktop/hosting-metadata.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Hosting metadata for software applications 
+subsection: desktop
+description: >
+  How to host your GUI desktop application's metadata for yum/DNF or Flatpak in self- or Github-hosted repositories
+---
+
+# Hosting metadata for yum/DNF or Flatpak
+
+Metadata for distribution tools and GNOME Software must be refreshed
+periodically, such as when new versions of [packaged
+applications](desktop-software.html) are released for consumption by
+users.  In many cases, Linux distributions take care of the metadata
+extraction, generation, and hosting for these purposes.  However, this
+section provides information on how to host your own metadata in case
+you need to do so.
+
+## Install helper utilities
+
+Install the following software to help parse and generate the metadata needed for hosting:
+
+```console
+$ sudo dnf install createrepo_c libappstream-glib-builder flatpak-builder
+```
+
+## Generating yum repository metadata
+
+1. Run these commands to create the initial metadata for source
```

Comment:
Why is the numbering here? It's a bit confusing, as the following one is also `1.`.

Reply:
These are steps in a procedure. If it's forbidden to number those, I'll leave them out. But using "1." for each step results in a properly numbered set of steps "1., 2., 3.," etc. without having to renumber manually in the event of a change. Is that not acceptable.

Reply:
Hmm, I did not know that. Does that work with GH-flavored MD? That's what jekyll uses to generate the resuilting website.

Reply:
I see what you mean by steps. However, like I said, this should be a complete guide, thus no steps are needed. You can however outline what's usually done and how, but this web is meant only as a reference pointing to the full-featured guides if needed.